dawn/scripts/inference.py

```diff
@@ -79,12 +79,13 @@ def main(steps):
     times = []
 
     with torch.inference_mode():
+        time_start = time.time()
         for pred in rollout(model, batch, steps=steps):
-            time_start = time.time()
             preds.append(pred.to("cpu"))
             time_end = time.time()
             print(f"Time for one step: {time_end - time_start}")
             times.append(time_start - time_end)
```

Comment:
```suggestion
            times.append(time_end - time_start)
```
Also we should fix this bit